app/src/main/java/edu/agh/susgame/front/model/graph/Node.kt

```diff
@@ -2,32 +2,53 @@ package edu.agh.susgame.front.model.graph
 
 import edu.agh.susgame.front.util.Coordinates
 
-open class Node(
+abstract class Node(
     val id: Int,
     val name: String,
     val position: Coordinates,
-)
+) {
+    abstract fun getInfo(): String
+}
 
 class Router(
     id: Int,
     name: String,
     position: Coordinates,
-    var bufferSize: Int,
-    var bufferCurrentPackets: Int = 0,
-) : Node(id, name, position)
+    private var bufferSize: Int,
+    private var bufferCurrentPackets: Int = 0,
+) : Node(id, name, position) {
+    override fun getInfo(): String { // fow now it's in string, later on this should be improved to something more accurate
+        return "Id: $id\nName: $name\nBufferSize: $bufferSize\nBufferCurrentPackets: $bufferCurrentPackets"
```

Comment:
Ja bym proponował:
- Dodać na sam początek tym node'a, a więc `ROUTER`, `HOST` itd.
- Nie wyświetlać `Id`, bo jest to w sumie bardziej mylące przez to, że istnieją efektywnie dwie numeracje
- Wszystko na polski
- Super by było przenieść to do `Translation.kt`. Możesz albo poszczególne słowa np. `const val BUFFER_SIZE = "Rozmiar bufora"`, albo jako metodę, która przyjmuje kilka argumentów i klei je w jednego stringa — jak wolisz

Reply:
przeniosłem wszystko do `Translation`, w przyszłości pewnie wszystko będzie w postaci jakichś zapełniających się pasków, suwaków itd, ale warto narazie żeby było porządnie